contract_manager/scripts/upgrade_evm_entropy_contracts.ts

```diff
@@ -27,6 +28,17 @@ const parser = yargs(hideBin(process.argv))
     },
   });
 
+// Override these URLs to use a different RPC node for mainnet / testnet.
+// TODO: extract these RPCs to a config file (?)
+const RPCS: Record<string, string> = {
+  "mainnet-beta": "https://api.mainnet-beta.solana.com",
+  testnet: "https://api.testnet.solana.com",
+};
+1;
+function registry(cluster: PythCluster): string {
+  return RPCS[cluster as string];
```

Comment:
you can remove the `as string` here and the `Record<string,string>` from the `const RPCS` to simplify the typing a bit.